Imagekit/Helper/TransformationTypes.cs

```diff
@@ -407,6 +218,18 @@ public Transformation EffectGray()
         return this.Add("e-grayscale", "true");
     }
 
+    /// <summary></summary>
+    public Transformation EffectShadow()
+    {
+        return this.Add("e-shadow", "true");
+    }
+
+    /// <summary></summary>
+    public Transformation EffectGradient()
+    {
+        return this.Add("e-gradient", "true");
+    }
+
```

Comment:
Please document these transformations also in Readme.md also

Reply:
done